content/dev-tools/utils/websocketapi-modals.tsx

```diff
@@ -0,0 +1,282 @@
+import * as React from "react";
+
+interface PermaLinkProps {
```

Comment:
nit: I'd recommend using either Permalink or PermanentLink, but not both for the names throughout this PR (And note that I think `Permalink` is one word not two)